src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -0,0 +1,157 @@
+import FormFieldThemeTopicSelect from '@admin/components/form/FormFieldThemeTopicSelect';
+import { Theme } from '@admin/services/themeService';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import { FormFieldset, FormFieldSelect } from '@common/components/form';
+import Form from '@common/components/form/Form';
+import FormFieldTextInput from '@common/components/form/FormFieldTextInput';
+import ModalConfirm from '@common/components/ModalConfirm';
+import useFormSubmit from '@common/hooks/useFormSubmit';
+import useToggle from '@common/hooks/useToggle';
+import { PublicationSummary } from '@common/services/publicationService';
+import { mapFieldErrors } from '@common/validation/serverValidations';
+import Yup from '@common/validation/yup';
+import { Formik } from 'formik';
+import React from 'react';
+
+export interface FormValues {
+  supersededById?: string;
+  title: string;
+  topicId: string;
+}
+
+interface Props {
+  canUpdatePublicationSupersededBy?: boolean;
+  canUpdatePublicationTitle?: boolean;
+  initialValues: FormValues;
+  publications?: PublicationSummary[];
+  themes?: Theme[];
+  onCancel: () => void;
+  onSubmit: (values: FormValues) => void;
+}
+
+const PublicationDetailsForm = ({
+  canUpdatePublicationSupersededBy = false,
+  canUpdatePublicationTitle = false,
+  initialValues,
+  publications,
+  themes,
+  onCancel,
+  onSubmit,
+}: Props) => {
+  const [showConfirmModal, toggleConfirmModal] = useToggle(false);
+
+  const errorMappings = [
+    mapFieldErrors<FormValues>({
+      target: 'title',
+      messages: {
+        SlugNotUnique: 'Choose a unique title',
+      },
+    }),
+  ];
```

Comment:
Typically error mappings are placed the module level. Other than preventing them from being re-constructed when the component re-renders, they also take up a lot of space and don't need to be within the component itself.